mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule Tirexs.Mixfile do
   use Mix.Project
 
   def project do
-    [ app: :tirexs, version: "0.7.5", elixir: "~> 1.2.0", description: description, package: package, deps: deps ]
+    [ app: :tirexs, version: "0.7.5", elixir: "~> 1.2.1", description: description, package: package, deps: deps ]
```

Comment:
We can't change dependency requirement in the middle of `v0.7.x` development. I'm pretty sure that many of `tirexs`users are still running elixir `v1.2.0`. (Also, it's OK for those who have been running the `v1.2.1`).